JosephNjuguna/WayFarerV2
# #167772560 admin create trip(database).

### What does this PR do?
create trips by admin

adds trip database table

update validations

update test files

### Description of the task to be completed?

- [x] enable admin to successfully create a trip

### How should this be manually tested?

in the .env file add 
```
required data is provided in the readme file
```

to run test : 
```
npm test
```
to test api on postman
```
npm start
```
console should log : 
```
 api started on port 3000
```
api endpoint : **POST**
```
/api/v1/trips
```
### Any background context you want to provide?
- [ ] testing data is provided in README
### Relevant pivotal tracker stories?
[#167772560](https://www.pivotaltracker.com/story/show/#167772560)

### Screenshots Available:
![Screenshot from 2019-08-11 23-21-44 - 1](https://user-images.githubusercontent.com/44730807/62839784-42022b00-bc98-11e9-8700-7d977d66961d.png)
